packages/node_modules/@node-red/nodes/core/network/22-websocket.html

```diff
@@ -40,6 +40,99 @@
 
 (function() {
 
+    const headerTypes = [
+        /*
+        { value: "Accept", label: "Accept", hasValue: false },
+        { value: "Accept-Encoding", label: "Accept-Encoding", hasValue: false },
+        { value: "Accept-Language", label: "Accept-Language", hasValue: false },
+        */
+        { value: "Authorization", label: "Authorization", hasValue: false },
+        /*
+        { value: "Content-Type", label: "Content-Type", hasValue: false },
+        { value: "Cache-Control", label: "Cache-Control", hasValue: false },
+        */
+        { value: "User-Agent", label: "User-Agent", hasValue: false },
+        /*
+        { value: "Location", label: "Location", hasValue: false },
+        */
+        { value: "other", label: RED._("node-red:httpin.label.other"),
+          hasValue: true, icon: "red/images/typedInput/az.svg" },
+    ]
+
+    const headerOptions = {};
+    const defaultOptions = [
+        { value: "other", label: RED._("node-red:httpin.label.other"),
+          hasValue: true, icon: "red/images/typedInput/az.svg" },
+        { value: "env", label: "env", hasValue: true },
```

Comment:
```suggestion
        "env",
```

This will cause it to use the built-in `env` type without having to replicate all the logic it provides - including auto-complete in 4.0.